source/code/plugins/in_dsc_monitor.rb

```diff
@@ -57,8 +57,8 @@ def run_check_install
 
     def get_dsc_status
       begin
-        python = %x(which python2)
-        if !python.empty?
+        python = (%x(command -v python3)).to_i
```

Comment:
Are you sure we can use `command` in ruby code? when i try it in `irb`, it is not found

```
situhan@lazarus:~/Git/Compute-Runtime-Tux/mdsdtest/fuzzing/listeners$ ruby -c "%x(command -v python3)"
Traceback (most recent call last):
ruby: No such file or directory -- %x(command -v python3) (LoadError)
```

Reply:
Yes, I wanted to make this code to use same path for checking python version as the shell script, but apparently, %x is not handling all system cmds same way. I will change it back to be with "which" command.